### PR TITLE
add canvas and input to game component

### PR DESCRIPTION
**Pull request by mJastrzebski6**
add canvas and text input to game component

### CODE REVIEW

File: src/components/GamePage.tsx [hunk 1]
```diff
@@ -6,7 +6,14 @@ function GamePage({changeStage}: GamePageInterface) {
 
     return (
         <div className="flex flex-col items-center gap-3 bg-gray-800 h-screen">
-        <button className="bg-red-600 hover:bg-red-700 text-white px-2 py-1 rounded text-xl" onClick={()=>changeStage(-1)}>LEAVE</button>
+            <button className="bg-red-600 hover:bg-red-700 text-white px-2 py-1 rounded text-xl" onClick={()=>changeStage(-1)}>LEAVE</button>
+            <div className="flex flex-row">
+                <canvas width={100} height={100}></canvas>
+                <div>   
+                    <div>Zgadywane teksty</div>
+                    <input type="text"></input>
```
Comment (AdamSzL) [inline src/components/GamePage.tsx:14]: We could add a placeholder to this input

**Review by AdamSzL — APPROVED** [review]
The code looks fine to me

**Comment by AdamSzL** [comment]
The feature was implemented and the pull request will be merged soon

File: src/components/GamePage.tsx (hunk 1 above)
Comment (mJastrzebski6) [reply src/components/GamePage.tsx:14]: you're right

**Pull request merged**